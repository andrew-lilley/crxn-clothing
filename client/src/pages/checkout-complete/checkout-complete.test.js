import React from 'react';
import { shallow } from 'enzyme';
import CheckoutCompletePage from './checkout-complete.component';

test('should render CheckoutCompletePage component', () => {
  const wrapper = shallow(<CheckoutCompletePage />);
  expect(wrapper).toMatchSnapshot();
});