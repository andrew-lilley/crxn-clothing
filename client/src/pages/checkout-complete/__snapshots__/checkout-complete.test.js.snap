// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CheckoutCompletePage component 1`] = `
<styled.div>
  <styled.h2>
    Order Complete
  </styled.h2>
  <p>
    Your payment has been successful.
  </p>
  <p>
    Thank you for placing your order with CRXN Clothing.
  </p>
</styled.div>
`;
